show human readable sizes in files

First of all, I want to thank you for this app!
My suggestion is related to Remote File Manager. Is it possible for the size of the files to appear in MB or GB? Even if I were a math genius, I still have to sit with a computer to calculate millions of bytes and convert them into MB. It would be useful for those who use the function.

@si458 
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t https://github.com/Ylianst/MeshCentral.git
npm ERR! enoent
npm ERR! enoent
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

The server is off but still get this error
c:\Meshcentral>npm install Ylianst/MeshCentral#display-file-sizes
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/Ylianst/MeshCentral.git
npm ERR! enoent
npm ERR! enoent
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

--- a/views/default.handlebars
+++ b/views/default.handlebars
@@ -862,6 +862,13 @@
                     <tr>
                         <td class="areaHead3">
                             <div class="toright2">
+                                <select id=p13sizedropdown onchange=p13updateFiles()>
+                                    <option value=0 selected="selected">Human Readble</option>
+                                    <option value=1 title=Bytes>Bytes</option>
+                                    <option value=10 title=KB>Kilobytes</option>
+                                    <option value=20 title=MB>Megabytes</option>
+                                    <option value=30 title=GB>Gigabyte</option>
+                                </select>
                                 <select id=p13sortdropdown onchange=p13updateFiles()>
                                     <option value=1 selected="selected">Sort by name</option>
                                     <option value=2>Sort by size</option>
@@ -10771,7 +10778,7 @@
 
         function p13updateFiles(checkedNames) {
             var html1 = '', html2 = '', displayPath = '<a href=# style=cursor:pointer onclick="return p13folderup(0)">' + "Root" + '</a>', fullPath = 'Root';
-
+            if(p13filetree==null) return;
             // Work on parsing the file path
             var x = p13filetree.path.split('\\');
             p13filetreelocation = [];
@@ -10798,7 +10805,22 @@
 
                 // Figure out the size
                 var fsize = '';
-                if (f.s != null) { fsize = getFileSizeStr(f.s); }
+                if (f.s != null) { 
+                    const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB'];
+                    const i = parseInt(Math.floor(Math.log(Math.abs(f.s)) / Math.log(1024)), 10);
+                    const option = Q('p13sizedropdown').options[Q('p13sizedropdown').selectedIndex];
+                    if(Q('p13sizedropdown').value==0){
+                        if (f.s === 0){
+                            fsize = 'n/a';
+                        }else{
+                            fsize = (i === 0 ? `${f.s} ${sizes[i]}` : `${(f.s / (1024 ** i)).toFixed(2)} ${sizes[i]}`);
+                        }
+                    }else if(Q('p13sizedropdown').value==1){
+                        fsize = getFileSizeStr(f.s);
+                    }else{
+                        fsize = `${(f.s / (2 ** option.value)).toFixed(2)} ${option.title}`;
+                    }
+                }
 
                 var h = '';
                 if (f.t < 3) {

